bugbug/bugzilla.py

```diff
@@ -514,9 +534,31 @@ def calculate_maintenance_effectiveness_indicator(
     print("After applying weights:")
     print(data)
 
+    weighed_open_defects = sum(data["open"].values())
     weighed_opened_defects = sum(data["opened"].values())
     weighed_closed_defects = sum(data["closed"].values())
+
     if weighed_opened_defects > 0:
-        return 100 * weighed_closed_defects / weighed_opened_defects
+        mei = 100 * weighed_closed_defects / weighed_opened_defects
+    else:
+        mei = 100 * (weighed_closed_defects + 1)
+
+    duration = (to_date - from_date).total_seconds() / 86400
```

Comment:
IIUC this will give us the burndown time in days. I assume this might look quite frustrating and proposed years in the document.

Reply:
Makes sense, I changed it to years.
In case we see for some teams it is too low, we could make it years for teams where it is at least a year, and months for the rest.